daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
+    return{
+        "statusCode": 200,
+        "body": json.dumps(response_string)
+    }
+
+
 def lambda_handler(event, context):
     person = people[get_random_index(len(people))]
     cause = causes[get_random_index(len(causes))]
+    message_string = f"Noted billionare {person} once again contributes nothing to {cause}."
+    if event.get("source", False) == "aws.events":
+        sendTweet(message_string)
+    else:
+        returnResponse(message_string)
 
-    sendTweet(f"Noted billionare {person} once again contributes nothing to {cause}.")
 
 if __name__ == "__main__":
     lambda_handler(None, None)
```

Comment:
If you take the suggestion up on line 66, this can just become:

```suggestion
    lambda_handler()
```